flow/Net2.actor.cpp

```diff
@@ -1315,6 +1311,11 @@ void Net2::run() {
 				TraceEvent(SevError, "TaskError").error(unknown_error());
 			}
 
+			if (currentTaskID < minTaskID) {
+				trackAtPriority(currentTaskID, taskBegin);
```

Comment:
Why do we move this tracking logic to after the task runs? 
What problem does it fix?

Reply:
I believe what's being addressed here is that a task can change its priority, and this causes us to evaluate the priority after its run and potentially done that adjustment.